config/locales/en/providers.yml

```
@@ -228,7 +228,7 @@ en:
           no_hint: Correspondence will be addressed to the client
     confirm_dwp_non_passported_applications:
       show:
-        tab_title: DWP records show that your client does not get a passporting benefit – Apply for Legal Aid - GOV.UK
+        tab_title: DWP records show that your client does not get a passporting benefit – Apply for Civil Legal Aid - GOV.UK
```

This seems to be a different case to the other examples of page title, but it's not a huge issue

---

The "Apply for Civil Legal Aid" bit specifically ^^

---

weird the tab_title should even be specified. It should, I'd hope be handled by one of the templates files and helpers. But it should be sentence case I pretty sure 


```suggestion
        tab_title: DWP records show that your client does not get a passporting benefit – Apply for civil legal aid - GOV.UK
```

---

I would remove the service name entirely as it is being duplicated - see `html_title` method

```suggestion
        tab_title: DWP records show that your client does not get a passporting benefit
```